jinahub/segmenters/VADSpeechSegmenter/tests/conftest.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+import os
+from typing import Callable
+from pathlib import Path
+
+from jina import Document, DocumentArray
+import numpy as np
+import torchaudio
+
+from ..vad_speech_segmenter import VADSpeechSegmenter
+
+
+@pytest.fixture(scope='module')
+def segmenter(tmpdir_factory) -> 'VADSpeechSegmenter':
+    workspace = tmpdir_factory.mktemp('data') / 'segmenter'
+    workspace.mkdir()
```

Comment:
Why not simply use `tmp_path` fixture - should be no need to create all these dirs?

Reply:
Using the `tmp_path` or `tmp_dir` fixure will raise the following error:
```
 You tried to access the 'function' scoped fixture 'tmp_path' with a 'module' scoped request object, involved factories
 ```
 The `segmenter` fixture is created in the `module` scope because I want to re-use it in multiple tests to save time. I have removed `workspace.mkdir()` to make this more concise.